glean-core/src/lib.rs

```diff
@@ -694,6 +696,20 @@ pub fn glean_test_get_experiment_data(experiment_id: String) -> Option<RecordedE
     core::with_glean(|glean| glean.test_get_experiment_data(experiment_id.to_owned()))
 }
 
+/// Sets a remote configuration for the metrics' disabled property
+///
+/// See [`core::Glean::set_metrics_disabled_config`].
+pub fn glean_set_metrics_disabled_config(json: String) {
+    match MetricsDisabledConfig::try_from(json) {
+        Ok(cfg) => launch_with_glean(|glean| {
+            glean.set_metrics_disabled_config(cfg);
+        }),
```

Comment:
Yes, in Desktop this _only_ happens after Glean is initialized, specifically right after FOG init is called in BrowserGlue.jsm.  After that, it could be called via the callback registered with Nimbus, but Glean should be initialized at that point.